Source/DafnyCore/Compilers/Python/PythonBackend.cs

```diff
@@ -46,6 +47,16 @@ public class PythonBackend : ExecutableBackend {
     return Path.GetExtension(externFilename) == ".py" ? Path.GetFileNameWithoutExtension(externFilename) : null;
   }
 
+  private static readonly Dictionary<OSPlatform, string> PlatformDefaults = new() {
+    {OSPlatform.Linux, "python3"},
+    {OSPlatform.Windows, "python"},
+    {OSPlatform.FreeBSD, "python3"},
+    {OSPlatform.OSX, "python3"},
+  };
+  private static string DefaultPythonCommand {
+    get => PlatformDefaults.Single(kv => RuntimeInformation.IsOSPlatform(kv.Key)).Value;
```

Comment:
It would be good to have a fallback value in case the platform value is not among the hardcoded list, instead of throwing an exception.

Reply:
Good call :P I felt since I exhausted the options from `OSPlatform` it wasn't much of an issue, but it can't hurt to use `SingleOrDefault` instead of `Single`.